README.adoc

```diff
@@ -57,12 +57,18 @@ Try to use at least version `0.11.0` as we haven't tested another version.
 Assuming you have Go installed and configured (have `$GOPATH` setup) here is
 how to build.
 
-Check out the code
+Check out the code. On Linux or Mac:
 
 ----
 $ git clone https://github.com/almighty/almighty-core $GOPATH/src/github.com/almighty/almighty-core
 ----
 
+On Windows PowerShell:
+
+----
+$ git clone https://github.com/almighty/almighty-core $env:GOPATH\src\github.com\almighty\almighty-core
+----
```

Comment:
Come on, there are so many more shells out there (even on linux) why should we support powershell? People know how to checkout stuff on windows using their tortoise tool. On windows you can also download the official git client from git-scm.com. Once installed there's a git bash which works just like on any linux and it handles command the same way.